src/platform/platform_linux.cpp

```diff
@@ -313,6 +314,39 @@ void mp::platform::Platform::remove_alias_script(const std::string& alias) const
         throw std::runtime_error(strerror(errno));
 }
 
+auto mp::platform::Platform::extra_daemon_settings() const -> SettingSpec::Set
+{
+    return {};
+}
+
+auto mp::platform::Platform::extra_client_settings() const -> SettingSpec::Set
+{
+    return {};
+}
+
+QString mp::platform::Platform::daemon_config_home() const // temporary
+{
+    auto ret = QString{qgetenv("DAEMON_CONFIG_HOME")};
+    if (ret.isEmpty())
+        ret = QStringLiteral("/root/.config");
+
+    ret = QDir{ret}.absoluteFilePath(mp::daemon_name);
+    return ret;
+}
```

Comment:
We could, and I started on that road but found that it would require much more than immediately apparent. This PR did not seem like the proper place for it.